third_party/python/Include/pymem.h

```diff
@@ -63,6 +64,11 @@ int _PyTraceMalloc_Untrack(
 PyObject* _PyTraceMalloc_GetTraceback(
     _PyTraceMalloc_domain_t domain,
     uintptr_t ptr);
+#else
+#define PyTraceMalloc_Track(domain, ptr, size) (-2)
```

Comment:
Did you forget the underscores here?

What you can do in this case is, something along the lines of:

```c
/* header */
int _PyTraceMalloc_Track(_PyTraceMalloc_domain_t, uintptr_t, size_t);
#ifdef MODE_DBG
#define _PyTraceMalloc_Track(domain, ptr, size) (-2)
#endif

/* implementation */
int (_PyTraceMalloc_Track)(_PyTraceMalloc_domain_t domain, uintptr_t ptr, size_t size) {
// ....
```

That's my favorite trick for overriding functions with a macro.